tests/unit/jobserver/views/test_job_requests.py

```diff
@@ -666,6 +667,7 @@ def test_jobrequestdetail_with_permission(rf, django_assert_num_queries):
 
     assert response.status_code == 200
     assert not response.context_data["is_invalid"]
+    assert response.context_data["is_missing_updates"]
```

Comment:
If I understand this correctly, we don't have a test for a job that's currently running on a backend that's been contacted recently (i.e not missing). Might be worth adding, just to check all the logic.